lib/backend/dynamo/dynamodbbk.go

```diff
@@ -65,15 +65,35 @@ type Config struct {
 	PollStreamPeriod time.Duration `json:"poll_stream_period,omitempty"`
 	// RetryPeriod is a period between dynamo backend retries on failures
 	RetryPeriod time.Duration `json:"retry_period"`
+
+	// EnableContinuousBackups is used to enable (or disable) PITR (Point-In-Time Recovery).
+	EnableContinuousBackups bool `json:"continuous_backups"`
+
+	// EnableAutoScaling is used to enable (or disable) auto scaling policy.
+	EnableAutoScaling bool `json:"auto_scaling"`
```

Comment:
Nit: All of these new fields are presumably optional, so I would consider adding `omitempty` to them.

Reply:
Done.